### DISCUSSION TITLE
File picker

### DISCUSSION THREAD

**Discussion by leolll** [wireui/wireui#167] [Q&A]
Just wonder if WireUI has a File Picker component? There's a nice mockup in the Modal page > Modal Card documentation, but it doesn't seem to be functional. 

https://livewire-wireui.com/docs/modal

**Comment by ph7jack** [comment]
@leolll There isn't yet, but I'm planning to create one soon.

**Reply by vlados** [reply]
This is a nice idea - https://pqina.nl/filepond/
There is a good implementation - https://laravel-form-components.randallwilk.dev/components/filepond
